ui/src/app/domains/views/DomainsList/DomainListHeaderForm/DomainListHeaderForm.tsx

```diff
@@ -13,7 +16,7 @@ type Props = {
   closeForm: () => void;
 };
 
-export type CreateDomain = {
+export type CreateDomainValues = {
```

Comment:
@ClementChaumel This is just to be consistent with other forms - forms get `Values` on the end, action parameters get `Params` on the end